pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --cov=graspy --doctest-modules
```

Comment:
I removed this from being *always* run because it was happening during CICD too.  It does make people have to opt into doing it on their own, but if it's useful I can put together a Makefile so you can just say `make test` and it'll wrap up all the commands we actually want to run.  It won't be useful to Windows, but at least on linux and mac we'll be in good shape.  Thoughts?

Reply:
CICD?

Reply:
I would certainly include `--doctest-modules` for testing. It tests the outputs in docstrings.

Reply:
@bdpedigo:
CICD -> Continuous Integration, Continuous Deployment.  Not entirely convinced we're doing the full thing but at least CI.  I just am used to saying them both!

@j1c:
Yep yep - it's still there!  I just didn't want to force coverage on all pytest runs since we're doing the coverage check quickly, early, in the build, and then NOT doing it on each of the 9 target environments (which turn into 12 soon enough, and conceivably 15 in a year or two).

Since coverage shouldn't be platform specific, I wanted treat them as atomic actions instead of a combined action :)

Reply:
yeah i am okay with making it atomic

Reply:
at least for the coverage part, doctest is maybe worth doing on all platforms